utils.sh

```diff
@@ -211,6 +211,25 @@ function sync_repo_and_patch {
     popd
 }
 
+function make_clouds_file {
+    local outdir="$1"
+
+    local BOOTSTRAP_IP=$(python -c "from ansible.plugins.filter import ipaddr; print(ipaddr.nthhost('"$PROVISIONING_NETWORK"', 2))")
+    local CLUSTER_IP=$(python -c "from ansible.plugins.filter import ipaddr; print(ipaddr.nthhost('"$PROVISIONING_NETWORK"', 3))")
+
+    cat - >${outdir}/clouds.yaml <<EOF
```

Comment:
One disadvantage to this approach is you have to change directory to access each cloud (AFAICS there's no CLI option to point to a clouds.yaml that's not in one of the default locations or $PWD?)

Would it be more user-friendly to instead write out a clouds.yaml in $PWD (e.g the base dev-scripts dir) and merge all the clouds into one, maybe renaming them to the deployed cluster name instead of "metal3" perhaps?

Reply:
That change would have to be in metal3-dev-env. I've been working on this for about a week now, and think this is incrementally better than what we have now. We can keep working on it in another PR.

Reply:
I should also say that the current behavior of having $PWD/clouds.yaml is intact, so users generating a single cluster won't notice this change.

Reply:
Ack sounds reasonable - perhaps we can add a j2 template and generate the clouds.yaml file via ansible or something but doing that as a followup seems fine, just wanted to mention it.